CONTRIBUTING.md

```diff
@@ -365,18 +370,103 @@ Instead, please:
 
 [Back to TOC](#table-of-contents)
 
+### Formatting Documentation
+
+The Kong docs site has a number of features built in to better display content.
+
+#### Markdown Front Matter
+
+Markdown files on the doc site (excluding `docs.konghq.com/hub/`) must have a
+yaml front matter section with at least one parameter (`title`). You can also
+specify additional parameters to change how your doc will display.
```

Comment:
"display" is a transitive verb that requires an object.  Either the fix is state the object, use passive, or say "renders" instead.

```suggestion
specify additional parameters to change how your doc is displayed.
```